website/src/docs/migration-guides.md

```diff
@@ -8,7 +8,66 @@ category: "Docs"
 
 These cover all the major Uppy versions and how to migrate to them.
 
-## Migrate to 2.0.0
+## Migrate from Uppy 2.x to 3.x
+
+### Uppy is pure ESM
+
+Following the footsteps of many packages, we now only ship [ECMAScript Modules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) (ESM). Uppy can’t be `require()`’d from CommonJS.
+
+Before Uppy 3.x, you could use `require` or `import` as we shipped CommonJS:
+
+<!-- eslint-disable no-restricted-globals -->
+
+```js
+const Uppy = require('@uppy/core')
+const Dashboard = require('@uppy/dashboard')
+```
+
+Now Uppy can only be `import`’ed:
+
+```js
+import Uppy from '@uppy/core'
+import Dashboard from '@uppy/dashboard'
+```
+
+Refer to the [Pure ESM package](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c) gist for added information and help.
+
+## Migrate from Companion 3.x to 4.x
+
+### Minimum required Node.js version is v14.19.0
+
+Aligning with the Node.js [Long Term Support (LTS) schedule](https://nodejs.org/en/about/releases/) and to use modern syntax features.
```

Comment:
Let's bump it to v14.20.0, as v14.19.0 has a rather serious security issue. Probably not something to address in this PR though, just commenting here to not forget.